docs/tutorial.md

```diff
@@ -58,12 +58,14 @@ var App = React.createClass({
     };
   },
   render() {
-    return <ul>{
-      // Access the data from trackMeteorData() on this.data
-      this.data.tasks.map(function (task) {
-        return <li key={task._id}>{task.content}</li>;
-      })
-    }</ul>
+    return (
+      <ul>
+        {/* Access the data from trackMeteorData() on this.data */}
+        {this.data.tasks.map(function (task) {
+          return <li key={task._id}>{task.content}</li>;
+        })}
```

Comment:
As far as I have seen this seems to be the more commonly used style, as blocks of javascript within jsx are "just syntactic sugar for function calls and object construction." https://facebook.github.io/react/tips/if-else-in-JSX.html also see https://facebook.github.io/react/docs/jsx-in-depth.html#comments